src/main/java/org/humancellatlas/ingest/project/ProjectService.java

```diff
@@ -182,31 +183,29 @@ private ProjectBag gather(Project project) {
         return new ProjectBag(projects, envelopes);
     }
 
-    public Page<Project> filterProjects(String search, String wranglingState, String wrangler, Pageable pageable) {
+    public Page<Project> filterProjects(SearchFilter searchFilter, Pageable pageable) {
         List<Criteria> criterias = new ArrayList<>();
         criterias.add(Criteria.where("isUpdate").is(false));
         Query query = new Query();
 
-        if (wranglingState != null) {
-            criterias.add(Criteria.where("wranglingState").is(wranglingState));
+        if (searchFilter.getWranglingState() != null) {
+            criterias.add(Criteria.where("wranglingState").is(searchFilter.getWranglingState()));
         }
 
-        if (wrangler != null) {
-            criterias.add(Criteria.where("primaryWrangler").is(wrangler));
+        if (searchFilter.getWrangler() != null) {
+            criterias.add(Criteria.where("primaryWrangler").is(searchFilter.getWrangler()));
         }
 
-        if (search != null) {
+        if (searchFilter.getSearch() != null) {
```

Comment:
You are calling `searchFilter.getSearch()` 4 times. Could you please create a local variable and use that in the code.
Tx